### DISCUSSION TITLE
set up custom routing

### DISCUSSION THREAD

**Discussion by numbereleven-a** [muink/luci-app-homeproxy#14] [General]
how to set up custom routing, what and where should I enter if I need a proxy only for certain nodes?
I want only youtube servers to use vpn

**Comment by muink** [comment]
Basically, it's not easy.
You need to write routing rules to let some traffic go through the proxy
Here are a few lists for reference, but you still need to convert to sing-box format yourself
https://github.com/LM-Firefly/Rules/blob/master/Global-Services/YouTube.list
https://github.com/ACL4SSR/ACL4SSR/blob/master/Clash/Ruleset/YouTube.list

There are also some simple [examples](https://github.com/muink/homeproxy-example)
